specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/playbookConfigurations.json

```
@@ -0,0 +1,461 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PlaybookConfigurationValidationStatus"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PlaybookConfigurationList": {
+      "description": "List of playbook configurations response.",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The list of playbook configurations under the given scope.",
+          "items": {
+            "$ref": "#/definitions/PlaybookConfiguration"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "PlaybookConfiguration": {
+      "type": "object",
+      "description": "The playbook configuration resource.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Playbook configuration data",
+          "$ref": "#/definitions/PlaybookConfigurationProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "PlaybookConfigurationProperties": {
+      "type": "object",
+      "description": "The playbook configuration data.",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The playbook configuration description."
+        },
+        "isEnabled": {
+          "type": "boolean",
+          "description": "Indicates whether the playbook configuration is enabled."
+        },
+        "metadata": {
+          "type": "object",
+          "description": "The metadata of the playbook configuration resource.",
+          "$ref": "#/definitions/PlaybookConfigurationMetadata"
+        },
+        "scopes": {
+          "type": "array",
+          "description": "A collection of the resources scopes on which the playbook configurations logic is applied.",
+          "items": {
+            "type": "string",
+            "description": "The resources scope."
+          }
+        },
+        "sources": {
+          "type": "array",
+          "description": "A collection of the source event types which evaluate the playbook configuration set of rules.",
+          "items": {
+            "$ref": "#/definitions/PlaybookConfigurationSource"
+          }
+        },
+        "actions": {
+          "type": "array",
+          "description": "A collection of the actions which are triggered upon true evaluation of the configured rule sets.",
```

The important part here is the "all"

```suggestion
          "description": "A collection of the actions which are triggered if all the configured rule set evaluation is true.",
```